themes/src/main/resources/theme/base/admin/resources/js/controllers/clients.js

```diff
@@ -929,10 +929,16 @@ module.controller('ClientDetailCtrl', function($scope, realm, client, flows, $ro
         {name: "INCLUSIVE_WITH_COMMENTS", value: "http://www.w3.org/TR/2001/REC-xml-c14n-20010315#WithComments"}
     ];
 
+    // KEYCLOAK-8460 Request Object Signature Verification Other Than RS256
     $scope.requestObjectSignatureAlgorithms = [
         "any",
         "none",
-        "RS256"
+        "RS256",
```

Comment:
See above. Do not hardcode list of algorithms.

Reply:
I'll do that the same way as in Token Signature SPI.